love-letter-app/src/app/globals.css

```diff
@@ -126,9 +107,97 @@ body {
 
 #myCard {
   transition: transform 0.3s ease;
-  
 }
 
 .flipped {
   transform: rotateY(180deg);
 }
+
+.creation-component {
+  display: flex;
+  flex-direction: column;
+  background-color: #fafafa;
+  width: 80%;
+  max-width: 800px;
+  margin: 20px;
+  height: 40%;
+  max-height: 323;
+  border-radius: 10px;
+  border: 1px solid black;
```

Comment:
let's use variable for black color if that's becoming a common theme for borders